_concepts/archived-search-functionality.md

```diff
@@ -35,6 +35,45 @@ tabs:
     below_image_content: What do you think of the Live Filters option?
     disqus_identifier: 35891276
     disqus_url: "http://treasury-dact-jekyll.s3-website-us-east-1.amazonaws.com/dev/concepts/search-functionality/#!live-filters"
+  - title: Advanced Search
+    url_hash: "#!advanced-search"
+    archived_text: The <a class='usa-archived-link usa-bold'
+    href='http://openbeta.usaspending.gov/concepts/search-results/index.html'>Search Results</a> concept has replaced this concept, which made it unnecessary.
```

Comment:
This line break is confusing Jekyll. Deleting it so that all the `archived_text` is on the same line will fix it. :)